contents/docs/getting-started/install.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Install PostHog
```

Comment:
I 1000% agree with this, although I think it's something that makes sense to tackle in a separate PR: basically, when you click on an SDK instead of taking you to a separate page it should just open the instructions inline.